README.md

```diff
@@ -36,11 +39,13 @@ In this version, the following DSC Resources are already built and ready for use
 - All base DSC resources found in PowerShell 5 ([WMF 5.0][wmf-5.0]).
 - All DSC resources found in the [Microsoft PowerShell DSC Resource Kit][DSCResources]
 
-## Windows System Prerequisites
+## [Windows System Prerequisites][wmf5-blog-requirements]
 
- - PowerShell 5, which is included in [Windows Management Framework 5.0][wmf-5.0]. PowerShell v5 is currently in [production preview][wmf5-blog-post], so the above link may change after official release.
- 
+ - PowerShell 5, which is included in [Windows Management Framework 5.0][wmf-5.0].
+ - Windows Server 2008 R2 SP1 - [WMF 4.0 and .NET Framework 4.5 or above are installed][wmf5-blog-requirements]
+ - Windows 7 SP1 - [WMF 4.0 and .NET Framework 4.5 or above are installed][wmf5-blog-requirements]
```

Comment:
Prerequisites for using this module require PowerShell 5 - I think you might be providing prerequisites for upgrading to WMF 5 here?